### ISSUE TITLE
Validation loss not decrasing

### ISSUE THREAD

**Issue by mit456** [willau/seq2seq-deep-spellchecking#1]
Hello, 

Thanks for sharing this great project, I am trying to retrain on news corpus and following your code, just a single change I have made, have increased vocab size (included caps chars and numbers) and have changed min-length to 60, but it seems after 1st epoch, the validation loss is not decreasing from `6.27`. I waited till 90th epoch but validation loss is the same. 
![screenshot from 2018-09-18 15-01-03](https://user-images.githubusercontent.com/4567139/45678328-c5cc9980-bb53-11e8-916e-b1225301de39.png)

Thanks.

**Comment by mit456** [comment]
Hello,

Can I know which version of keras are you using?
